From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 13 Jun 2013 10:42:58 +0200
Subject: [PATCH] add more search paths

Without this setup.py won't find libs in <sysroot>/lib

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 setup.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 66a97bd..7bd9e9c 100644
--- a/setup.py
+++ b/setup.py
@@ -484,8 +484,10 @@ class PyBuildExt(build_ext):
             # (PYTHONFRAMEWORK is set) to avoid # linking problems when
             # building a framework with different architectures than
             # the one that is currently installed (issue #7473)
-            add_dir_to_list(self.compiler.library_dirs,
-                            sysconfig.get_config_var("LIBDIR"))
+            libdir = sysconfig.get_config_var("LIBDIR")
+            add_dir_to_list(self.compiler.library_dirs, libdir)
+            if libdir.endswith('/usr/lib'):
+                add_dir_to_list(self.compiler.library_dirs, libdir.replace('/usr/lib','/lib'))
             add_dir_to_list(self.compiler.include_dirs,
                             sysconfig.get_config_var("INCLUDEDIR"))
 
